File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        m=len(accounts)
        n=len(accounts[0])
        mx=0
        for i in range(0,m,1):
            w=0
            for j in range(0,n,1):
                w+=accounts[i][j]
            if w> mx:
                mx =w
        
        return mx
            

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.rs
impl Solution {
    pub fn maximum_wealth(accounts: Vec<Vec<i32>>) -> i32 
    {
        let m=accounts.len();
        let n=accounts[0].len();
        let mut mx=0;
        for i in 0..m
        {
            let mut wealth =0;
            for j in 0..n
            {
                wealth+=accounts[i][j];
            }
            if(wealth>mx){mx=wealth;}
        }
        return mx;
    }
}